src/test/java/io/github/perrymant/moneymaker/FileIOTest.java

```diff
@@ -0,0 +1,29 @@
+package io.github.perrymant.moneymaker;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class FileIOTest {
+
+    @Test
+    public void canReadData()  {
```

Comment:
We're verifying that we're `fileIO.read()` in the other test too so maybe this test can go away?